src/content/en/2021/markup.md

```diff
@@ -82,7 +82,7 @@ In terms of popularity, HTML5, better known as `<!DOCTYPE html>` is still the mo
   ) }}</figcaption>
 </figure>
 
-The surprising part is that, <a hreflang="en" href="https://en.wikipedia.org/wiki/XHTML">almost 20 years later</a>, XHTML is still a considerable part of the web, with 8% of pages still using it on desktop and a little under 7% on mobile.
+The surprising part is that, [almost 20 years later](https://en.wikipedia.org/wiki/XHTML), XHTML is still a considerable part of the web, with 8% of pages still using it on desktop and a little under 7% on mobile.
 
 ### Document size
```

Comment:
Should this be translated?

Reply:
Isn't this your commitment?
https://github.com/HTTPArchive/almanac.httparchive.org/pull/2741/commits/718b5b6dca0946eae798c2ee9feee00ada52c190

Reply:
I was talking about the “Document size” title for this comment. I fixed the nearby link in that other commit but still think the title is worthy of translating. What do you think?

Reply:
Sorry, I was wrong.
I translated. Thanks